packages/api/models/schemas.js

```diff
@@ -255,15 +255,13 @@ module.exports.granule = {
         type: 'object',
         properties: {
           bucket: { type: 'string' },
```

Comment:
These aren't marked as required because the granules are written to the database at the start of a workflow, before a lot of that information is available.